packages/frontend-shared/src/warning/Warning.vue

```diff
@@ -68,3 +69,12 @@ let message = computed(() => {
 
 const { markdown } = useMarkdown(markdownTarget, message.value, { classes: { code: ['bg-warning-200'] } })
 </script>
+
+<style lang="scss">
+.warning-markdown {
```

Comment:
Done - I still need to do `.warning-markdown > ul` since you cannot nest selectors using Tailwind, but I changed it to use `@apply`.

The reason I did it like this is we need to style the `<ul>` which is part of the generated HTML in the `errors` package. I tried adding it with the [rest of the styles](https://github.com/cypress-io/cypress/blob/develop/packages/frontend-shared/src/composables/useMarkdown.ts#L46) but it's getting clobbered by something else and the styles aren't applied:
![image](https://user-images.githubusercontent.com/19196536/181689832-97b69ab1-94ad-47fc-ad47-0ca5b4d253a9.png)